### ISSUE TITLE
Feature Request: Multi-account support

### ISSUE THREAD

**Issue by tandy-1000** [ajbura/cinny#66]
#### Is your feature request related to a problem? Please describe.
Hydrogen is the only other web client which offer multi-account support, it would be nice to have a more modern client with this feature.

#### Describe the solution you'd like
Hydrogen does it like this:
![image](https://user-images.githubusercontent.com/24867509/129892464-29ce4c55-752b-4fcb-9a12-b868fb9ea4c0.png)

**Comment by viasux** [comment]
Would love this! <3 - Also if it could be multi account across other homeservers as well!~

**Comment by redthing1** [comment]
This would be awesome.

**Comment by kfiven** [comment]
We have some underlying support done for multiple accounts but this is on hold until we figure out how multiple accounts are going to work with OIDC as overall account management is no longer a client's job.

**Comment by olfMombach** [comment]
We're using Matrix at work, but I don't want to join with my personal account, so this feature would be very helpful to have.

Edit: As a workaround for now, Firefox's built in "Multi-Account Containers" extension works very well with this. Open Cinny in different Containers and you can have it running in parallel with different accounts.

**Comment by yoobi** [comment]
Hello any eta for this or what could someone do to help ?

**Comment by trymeouteh** [comment]
Would like this as well with the ability to recieve notification from all logged in accounts

**Comment by kfiven** [comment]
Hi, This feature is planned in next release. @yoobi

**Comment by claxonberlin** [comment]
That's huge news! I deeply believe client support for multiple accounts across homeservers is the path to more widespread professional adoption of matrix. A true cross-ecosystem e-chat (like e-mail) is going to be fantastic.
